tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -391,14 +394,35 @@ in V is very simple:
 import vweb
 import json
 
+['/articles'; get]
 pub fn (mut app App) articles() vweb.Result {
 	articles := app.find_all_articles()
 	return app.json(json.encode(articles))
 }
 ```
 
+
 <img width=662 src="https://github.com/vlang/v/blob/master/tutorials/building_a_simple_web_blog_with_vweb/img/articles_json.png?raw=true">
 
+### Persistent data
+If one wants to persist data they need to use a file instead of memory SQLite Database.
+Replace the `init_server()` function with this instead:
+
+```v oksyntax
+pub fn (mut app App) init_server() {
+	app.db = sqlite.connect('blog.db') or { panic(err) }
+	app.db.create_table('Article', [
+		'id integer primary key',
+		"title text default ''",
+		"text text default ''",
+	])
+}
+```
+
+As we can see it attempts to connect to a file in the active pathcalled `blog.db`. 
```

Comment:
attemps to open a database file in the current path named `blog.db`

Reply:
updated